sdcm/cql_stress_cassandra_stress_thread.py

```diff
@@ -0,0 +1,158 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2023 ScyllaDB
+
+import logging
+import re
+import os
+import contextlib
+from sdcm.loader import CqlStressCassandraStressExporter
+from sdcm.prometheus import nemesis_metrics_obj
+from sdcm.sct_events.loaders import CqlStressCassandraStressEvent
+from sdcm.stress_thread import CassandraStressThread
+from sdcm.utils.common import SoftTimeoutContext
+from sdcm.utils.docker_remote import RemoteDocker
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+class CqlStressCassandraStressThread(CassandraStressThread):
```

Comment:
Why not `CqlStressThread` ?

Reply:
also big portion of `CassandraStressThread` implementation is copied here, 
so maybe there no reason to inherit from it at all ?

if we do inherit, we should move the relevant part which needs to be different to their own functions/properties that can be overridden

Reply:
> Why not `CqlStressThread` ?

As `cql-stress` supports two frontends (`cassandra-stress` and `scylla-bench`), I preferred to be more specific.